invenio_communities/members/services/config.py

```diff
@@ -37,6 +39,26 @@ class PublicSearchOptions(SearchOptions):
         ),
     }
 
+    query_parser_cls = QueryParser.factory(
+        # fields serves also as a filter, since only the ones mentioned there
+        # will be queried by ES, using multi_match query.
+        fields=[
```

Comment:
basically search will **only** happen over username, full_name and affiliations.